lms/resources/_js_config/__init__.py

```diff
@@ -214,6 +214,10 @@ def add_deep_linking_api(self):
                 "content_item_return_url": self._context.lti_params[
                     "content_item_return_url"
                 ],
+                "context_id": self._context.lti_params["context_id"],
```

Comment:
Send the context id back to ourselves. We could use `course.id` but I think that's a bit more confusing.